python/point_cloud2.py

```diff
@@ -357,6 +357,120 @@ def make_radar_point_cloud(header: Header, points: Iterable, is_dense: bool = Tr
     return create_cloud(header, fields, points, is_dense)
 
 
+def create_entities_cloud(
+        header: Header,
+        fields: Sequence,
+        entities: Iterable) -> PointCloud2:
+    """
+    Create a special provizio_dds.PointCloud2 message for entities.
+
+    :param header: The point cloud header. (Type: provizio_dds.Header)
+    :param fields: The entities point cloud fields. (Type: Sequence of provizio_dds.PointField)
+    :param entities: The entities to be sent. List of iterables, i.e. one iterable
+                   for each entity, with the elements of each iterable being the
+                   values of the fields for that entity (in the same order as
+                   the fields parameter)
+    :param is_dense: True if there are no invalid points
+    :return: The point cloud as provizio_dds.PointCloud2
+    """
+    entities_cloud = create_cloud(header, fields, entities)
+
+    return entities_cloud
+
+
+def make_radar_entities(
```

Comment:
@ClementBotty Please indeed also add an additional test to `test/python/pointcloud2_test.py`

Reply:
I added tests to the repository, to check metadata and data for embedded entities in PointCloud2.